Bottom: 8068b91b6e0211da0600fe0b8a80e030ad4de21e
Top:    452f2d770571959facd82a7a5574ce40a3665c58
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-18 20:05:00 -0700

Refresh of more-reset

---
diff --git a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
index fcf7f3ce70a..421e8af3517 100644
--- a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
+++ b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
@@ -188,7 +188,10 @@ class CStringArg {
 public:
   CStringArg(const char *s) { cstr = s; }
   CStringArg(const std::string &s) { cstr = s.c_str(); }
-  CStringArg(const llvm::Twine &twine) {
+
+  template <typename T>
+  CStringArg(std::enable_if<std::is_convertible<T, llvm::Twine>::value, T> x) {
+    llvm::Twine twine(x);
     llvm::StringRef ref = twine.toNullTerminatedStringRef(storage);
     cstr = ref.data();
   }
